sagemaker-inference-recommender/python-sdk-inference-recommender/inference-recommender-with-python-sdk.ipynb

```diff
@@ -0,0 +1,652 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">'OMP_NUM_THREADS': CategoricalParameter(['1', '2', '3', '4'])
</pre><p>triggers error:</p><pre class="ql-syntax" spellcheck="false">ClientError: An error occurred (ValidationException) when calling the CreateInferenceRecommendationsJob operation: 2 validation errors detected: Value '["1", "2", "3", "4"]' at 'inputConfig.endpointConfigurations.1.member.environmentParameterRanges.categoricalParameterRanges.1.member.value' failed to satisfy constraint: Member must have length less than or equal to 3; Value '["1", "2", "3", "4"]' at 'inputConfig.endpointConfigurations.2.member.environmentParameterRanges.categoricalParameterRanges.1.member.value' failed to satisfy constraint: Member must have length less than or equal to 3
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3767/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>